lib/hexpm/accounts/recovery_code.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Hexpm.Accounts.RecoveryCode do
+  use Hexpm.Schema
+
+  alias Hexpm.Accounts.RecoveryCode
+
+  @derive {Jason.Encoder, only: []}
+
+  @rand_bytes 10
+  @part_size 4
+  @typep recovery_code() :: <<_::152>>
+
+  @primary_key false
+  embedded_schema do
+    field :code, :string
+    field :used_at, :utc_datetime_usec
+  end
+
+  def changeset(recovery_code, params) do
+    cast(recovery_code, params, [:code, :used_at])
+  end
+
+  @spec gen_code_set() :: [%RecoveryCode{}]
+  def gen_code_set, do: Enum.map(1..5, fn _ -> %RecoveryCode{code: gen_code()} end)
```

Comment:
should we increase this to say 10? (GitHub generates 16)

```suggestion
  def gen_code_set, do: Enum.map(1..10, fn _ -> %RecoveryCode{code: gen_code()} end)
```